app/models/model.py

```diff
@@ -204,15 +55,83 @@ def check_num_items_holes(cls, v):
         for pt_list in v:
             for pt in pt_list:
                 if len(pt) != 3:
-                    raise ValidationError(
+                    raise ValueError(
                         'Number of floats must be 3.'
                     )
         return v
 
 
-class Ground(BaseModel):
+class ShadeEnergyPropertiesAbridged(BaseModel):
```

Comment:
@chriswmackey is there a `ShadeEnergyProperties` that is not abridged?

Reply:
@macumber ,
Yes, there is because `transmittance_schedule` is a property under `ShadeEnergyProperties`.  When you get a complete `ShadeEnergyProperties` dict, it includes the entire schedule under it so that it can completely re-serialize the original object.  However, when you find such shade properties in a `Model`, it exists as `ShadeEnergyPropertiesAbirdged` and you will only find the name of the transmittance schedule under it (the full definition of the transmittance schedule will go under a `schedules` key of the `ModelEnergyProperties`). For reasons like this, we are using the word `Abridged` under many of these energy property schemas.